pylint_django/compat.py

```diff
@@ -24,5 +24,7 @@
 
 import pylint
 
-# pylint before version 2.3 does not support load_configuration() hook.
-LOAD_CONFIGURATION_SUPPORTED = pylint.__pkginfo__.numversion >= (2, 3)
+# The __pkginfo__.numversion attribute was removed in pylint >= 2.8.0
+_pylint_version = pylint.__pkginfo__.version if hasattr(pylint.__pkginfo__, "version") else pylint.__pkginfo__.numversion
```

Comment:
CI still failing, not sure why it tries to access `numversion`. Maybe try `pylint.__version__`.